### PR TITLE
[master] Add byte sizing maximum defaults

### PR DESCRIPTION
**Pull request by henri-tremblay**

### CODE REVIEW

File: docs/src/docs/asciidoc/user/tiering.adoc [hunk 1]
```diff
@@ -82,7 +82,8 @@ include::{sourcedir33}/impl/src/test/java/org/ehcache/docs/Tiering.java[tag=byte
 There is a cost associated to sizing objects.
 <2> The settings are only used by the heap tier. So off-heap won't use it at all.
 <3> The sizing can also be further restrained by 2 additional configuration settings:
-The first one specifies the maximum number of objects to traverse while walking the object graph, the second defines the maximum size of a single object.
+The first one specifies the maximum number of objects to traverse while walking the object graph (default: 1000), the second
+defines the maximum size of a single object (default: unlimited).
```
Comment (ljacomet) [inline docs/src/docs/asciidoc/user/tiering.adoc]: Is it not `Long.MAX_VALUE`?

Comment (ljacomet) [inline docs/src/docs/asciidoc/user/tiering.adoc]: Let's be truthful saying what the limit is and adding that in practice it means unlimited.

Comment (henri-tremblay) [inline docs/src/docs/asciidoc/user/tiering.adoc]: done

Comment (ljacomet) [inline docs/src/docs/asciidoc/user/tiering.adoc]: I do not see that change.

Comment (henri-tremblay) [inline docs/src/docs/asciidoc/user/tiering.adoc]: Refresh. I do see it.

**Pull request merged**